requirements/tox_requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Packages needed in order to run tox.
+# Installed by Travis and dev environments.
+tox==2.5.0
```

Comment:
For consistent test builds, we usually try to pin indirect dependencies (like `py` for `tox`) to specific versions also.  We're starting to use `pip-tools` in a few repos to automate that for us.